src/test/resources/hw2ex1.xml

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="TaskOneSuite" verbose="1" >
+    <test name="IndexPageAndFrame" >
+        <classes>
+            <class name="com.epam.tc.hw2.ex1.ExerciseOne" />
+        </classes>
+    </test>
+</suite>
```

Comment:
Add a new blank line to the end of the file. Have you tried to run your HW via mvn command?